app/controllers/claims_controller.rb

```diff
@@ -297,4 +304,18 @@ def save_details_check
     details_check = params.dig(:claim, :details_check)
     DfeIdentity::ClaimUserDetailsCheck.call(current_claim, details_check)
   end
+
+  def update_session_with_tps_school
+    if (school = current_claim.recent_tps_school)
+      session[:tps_school_id] = school.id
+      session[:tps_school_name] = school.name
+      session[:tps_school_address] = school.address
+    end
+  end
+
+  def check_correct_school_params
+    updated_claim_params = CorrectSchoolForm.extract_params(claim_params, change_school: params[:change_school])
```

Comment:
Maybe the method should be `extract_attributes` instead of `extract_params`.

And we assign to `updated_claim_attributes`, or `updated_attributes`, `claim_attributes`. I think any of these three names would work.

WDYT?

Reply:
We can always do this in a separate PR.